.github/workflows/release.yml

```diff
@@ -49,7 +51,9 @@ jobs:
           python -m pip install -r requirements_dev.txt
           cd python
           git fetch origin master
-          TAICHI_CMAKE_ARGS=$CI_SETUP_CMAKE_ARGS python build.py build
+          export TAICHI_CMAKE_ARGS=$CI_SETUP_CMAKE_ARGS
+          [ -z "${{ github.event.action }}" ] && python build.py build --project_name taichi_nightly
```

Comment:
Nit: could you use if-else here instead of &&/||? IIUC here you just want to make sure there's only one wheel built, but it's less straightforward writing this way :P

Reply:
I tested using this syntax. And I am not sure about using if-else syntax. I mean this syntax looks a lot more simple and tidy? I can add a few comments to make this clearer?

Reply:
haha sure, adding a comment sounds good to me.